src/stats.rs

```diff
@@ -72,6 +74,8 @@ pub struct PeerStats {
     pub bwe_tx: Option<Bitrate>,
     /// The egress loss over the last second.
     pub egress_loss_fraction: Option<f32>,
+    /// The ingress losss since the last stats event.
```

Comment:
There's a discrepancy between the time period between egress and ingress loss now. We could address this by making ingress loss also emit every second to match egress.

For egress using data since last event is trickier when the stats interval is small, there's a risk it exceeds RTT and we have no data